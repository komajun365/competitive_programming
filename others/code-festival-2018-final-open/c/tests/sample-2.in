4
12 5
1 1
7 3
243 32
6
632
188
69
54
14
36
