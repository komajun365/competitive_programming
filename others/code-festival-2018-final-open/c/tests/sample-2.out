421
32
32
32
7
29
